manuscripts/method/03.results.md

```diff
@@ -9,76 +9,110 @@ MS-based global phosphoproteomic data provides unparalleled coverage when interr
+
+<!-- ERK2 substrates -->
+Next, we extracted the sequences of ERK2 substrates identified in Carlson et al to create an ERK2-specific PSSM positive control (Figure. {@fig:val}F). As expected, ERK2 was predicted to be the upstream kinase with the highest preference for the cluster's motif (Figure. {@fig:val}G). Finally, given the consistent enrichment of hydrophobic and polar residues throughout the entire ERK2 target motif (Figure. {@fig:val}F), we asked whether randomly shuffling all PSSM positions except the phosphoacceptor residue would affect the upstream kinase prediction. Interestingly, this synthetic motif led to a 2-fold less favorable motif for ERK2 (Figures. {@fig:val}G and H). Furthermore, we included those clusters from the CPTAC data set that were preferentially favored by ERK2 and performed the same experiment. As expected, we observed very similar trends (Figures. {@fig:val}G and H). Altogether, these results illustrate two different scenarios in which the cluster sequence information was successfully utilized to correctly identify upstream kinases.
+
+<!-- Notes figureM2:
+1. C is a more natural position than in D for the heatmap (first show data via PCA and heatmap A-C, then talk about prediction and substreates D-E). This means that the heatmap should be vertical and occupy slots C and E which I don't know how to code. I'll just edit on adobe once we're done reviewing.
+
+2. I'm showing kinase predictions of CPTAC clusters in C and G so that's why I'm introducing this data but I'm not sure if it's smooth enough (3rd sentence 1st paragraph). 
+
+3. Statistical test in AKT substrates plot?
```

Comment:
To help with this, can this be the actual count, not normalized?

Reply:
After that you can figure out significance using the cdf of a [hypergeometric distribution](https://en.wikipedia.org/wiki/Hypergeometric_distribution).